packages/react-magma-dom/src/components/Stepper/Stepper.tsx

```diff
@@ -0,0 +1,313 @@
+  }
+}
+
+const StyledStepper = styled.div<StepperProps>`
+  display: flex;
+  flex: 1;
+  flex-direction: column;
+  padding: ${props =>
+    props.summaryView ||
+    props.hideLabels ||
+    (props.hasBreakpoint && BreakPointStyle.summaryView) ||
+    (props.hasBreakpoint && BreakPointStyle.hideLabels)
+      ? ''
+      : '0 40px'};
+`;
+
+const StyledStepContent = styled.div<StepperProps>`
+  display: flex;
+`;
+
+const StyledSeparator = styled.div<StepperProps>`
+  background: ${StepSeparatorBackgroundColors};
+  width: 100%;
+  height: 2px;
+  top: 12px;
+  position: relative;
+  align-self: baseline;
+  transition: all 0.4s ease;
+`;
+
+const StyledSummary = styled.div<StepperProps>`
+  display: flex;
+  flex-direction: column;
+  top: 14px;
+  font-size: ${props => props.theme.typeScale.size02.fontSize};
+  color: ${props =>
+    props.isInverse
+      ? transparentize(0.3, props.theme.colors.neutral100)
+      : props.theme.colors.neutral500};
+  div > div {
+    margin: 4px 0 0 0;
+  }
+  span {
+    display: flex;
+    text-align: left;
+  }
+`;
+
+// Stepper!
+export const Stepper = React.forwardRef<HTMLDivElement, StepperProps>(
+  (props, ref) => {
+    const {
+      breakpoint,
+      breakpointStyle,
+      children,
+      currentStep,
+      stepDescriptionLabel,
+      isInverse: isInverseProp,
+      hideLabels,
+      summaryView,
+      testId,
+      ...rest
+    } = props;
+    const theme = React.useContext(ThemeContext);
+    const isInverse = useIsInverse(isInverseProp);
+
+    let [breakPointHideLabels, setBreakPointHideLabels] = React.useState(false);
+    let [breakPointSummaryView, setBreakPointSummaryView] =
+      React.useState(false);
+
+    // Breakpoint states when a specific breakpoint number is set, then a breakpointStyle which results in the removal of all text or the summary view.
+    const [windowWidth, setWindowWidth] = React.useState(window.innerWidth);
+
+    const hasBreakpoint = breakpoint > windowWidth;
+    const beyondBreakpoint = breakpoint < windowWidth;
+
+    React.useEffect(() => {
+      function handleResize() {
+        setWindowWidth(window.innerWidth);
+      }
+      window.addEventListener('resize', handleResize);
+      handleResize();
+
+      if (hasBreakpoint && breakpointStyle === BreakPointStyle.summaryView) {
+        setBreakPointSummaryView(true);
+      } else if (
+        hasBreakpoint &&
+        breakpointStyle === BreakPointStyle.hideLabels
+      ) {
+        setBreakPointHideLabels(true);
+      } else if (
+        (beyondBreakpoint && breakpointStyle === BreakPointStyle.hideLabels) ||
+        (beyondBreakpoint && BreakPointStyle.summaryView)
+      ) {
+        setBreakPointSummaryView(false);
+        setBreakPointHideLabels(false);
+      }
+
+      return () => window.removeEventListener('resize', handleResize);
+    }, []);
+
+    // Sets aria-current based on the active step, else false.
+    const ariaCurrent = index => {
```

Comment:
This can be done in one line:
```
const ariaCurrent = index => currentStep === index;
```

If you don't use it anywhere else, do this in place instead of creating a constant.